.github/workflows/release.yml

```
@@ -0,0 +1,123 @@
+name: 'Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version'
+        type: string
+        required: true
+
+jobs:
+  prepare:
+    runs-on: [ self-hosted ]
```

```suggestion
    runs-on: [ ubuntu-22.04 ]
```
we cannot use self-hosted runner for public repo

---

Updated